float32 cellSize
float32 originX
float32 originY
amee/ExploringGridColumn[] exploringGrid